examples/rbd/groupsnapshotclass.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: groupsnapshot.storage.k8s.io/v1alpha1
```

Comment:
@nixpanic, I see example values are openshift specific.
Can you make it similar to cephfs groupsnapshotclass.yaml example file?

Reply:
ah, yes, it seems I missed some changes for that

Reply:
corrected this, please check again, thanks!